plugins/processors/awsentity/factory.go

```diff
@@ -26,7 +26,9 @@ func NewFactory() processor.Factory {
 	return processor.NewFactory(
 		TypeStr,
 		createDefaultConfig,
-		processor.WithMetrics(createMetricsProcessor, stability))
+		processor.WithMetrics(createMetricsProcessor, stability),
+		processor.WithLogs(createLogsProcessor, stability),
```

Comment:
Added these in factory code so `awsentity` can be added as a log type processor but it doesn't really do anything now